src/NuGet.Clients/NuGet.CommandLine/Commands/SourcesCommand.cs

```diff
@@ -36,10 +32,7 @@ public class SourcesCommand : Command
         [Option(typeof(NuGetCommand), "SourcesCommandFormatDescription")]
         public SourcesListFormat Format { get; set; }
 
-        [Option(typeof(NuGetCommand), "SourcesCommandTrustDescription")]
```

Comment:
How about you jsut remove this option, and set the default to false and call it good? 

That way you don't run into merge conflicts when trying to bring it back later.

Reply:
discussed offline. it is cleaner to revert and then get it flow in along with a feature branch.